management-api-server/pom.xml

```diff
@@ -2,14 +2,6 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
 
-    <repositories>
-        <repository>
-            <id>jcenter</id>
-            <name>jcenter</name>
-            <url>https://jcenter.bintray.com/</url>
-        </repository>
-    </repositories>
-
```

Comment:
It looks like removing this repo is causing the CI build to fail.

https://github.com/datastax/management-api-for-apache-cassandra/actions/runs/636195617

```Could not find artifact com.aries:docker-java-shaded:jar:3.1.1 in central (https://repo.maven.apache.org/maven2)```

We either need to add it back, or adjust how the settings.xml file is constructed in the CI here:

https://github.com/datastax/management-api-for-apache-cassandra/blob/master/.github/workflows/ci.yaml#L48-L63

Reply:
Marking this resolved for now, but we'll have to address #93 at some point.